avalon/nuke/lib.py

```diff
@@ -24,6 +32,25 @@ def maintained_selection():
             [n['selected'].setValue(True) for n in previous_selection]
 
 
+def reset_selection():
```

Comment:
Maybe `deselect_all` is a better name?

Reply:
Yeah, @jezscha  what do you think ?

Reply:
Hmm, no wait. Since this isn't a new function that introduced here and already been used, so we better not to change the name. :thinking:

Reply:
Hi @davidlatwe sorry for the slow response.. Yes please this is used many times.. anyway you could rename it but link this for backward compatibility

Reply:
I can live with current name :)